Jenkinsfile

```diff
@@ -39,6 +39,12 @@ pipeline {
             description: 'Allow dynamic versions of dependencies (support projects without lock files)'
         )
 
+        booleanParam(
+            name: 'USE_CLEARLY_DEFINED_CURATIONS',
+            defaultValue: true,
+            description: 'Use package curation data from the <a href="https://clearlydefined.io/">ClearlyDefine</a> service'
```

Comment:
ClearleyDefine**d**: missing a `d`

Reply:
Done. FYI, I had to remove the link as it does not render as a link in Jenkins by default.

Reply:
Thanks!